script.js

```diff
@@ -140,7 +140,7 @@ const createButton = (text, isDisabled = false, type = 0) => {
   const button = document.createElement("button");
   button.textContent = text;
   button.disabled = isDisabled;
-  button.classList.toggle("today", isToday);
+  if(!button.disabled) button.classList.toggle("today", isToday);
```

Comment:
Thanks for spotting the bug and submitting a fix! I have a slight adjustment. Could you update your pull request with this change?
```js
button.classList.toggle("today", isToday && !isDisabled);
```

Reply:
Done

Reply:
Merged. Thanks for the fix! 🙏